Description: Disable volume-test
 It seems to be faulty and failing, even using pure upstream releases.
Author: Daniel van Vugt <daniel.van.vugt@canonical.com>
Bug: https://gitlab.freedesktop.org/pulseaudio/pulseaudio/issues/559
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1785556
Forwarded: no
Last-Update: 2018-08-06

diff --git a/src/Makefile.am b/src/Makefile.am
index f4464d253..4bc2590c0 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -269,7 +269,6 @@ TESTS_default = \
 		resampler-test \
 		smoother-test \
 		thread-test \
-		volume-test \
 		mix-test \
 		proplist-test \
 		cpu-mix-test \
